launcher/package/ubuntu/multimc/usr/share/man/man1/multimc.1

```diff
@@ -28,20 +28,20 @@
 .\" * MAIN CONTENT STARTS HERE *
 .\" -----------------------------------------------------------------
 .SH "NAME"
-multimc \- a launcher and instance manager for Minecraft\&.
+ultimmc \- a launcher and instance manager for Minecraft\&.
 .SH "SYNOPSIS"
 .sp
 \fBmultimc\fR [\fIOPTIONS\fR]
 .SH "DESCRIPTION"
 .sp
-MultiMC is a custom launcher for Minecraft that allows you to easily manage multiple installations of Minecraft at once\&. It also allows you to easily install and remove mods by simply dragging and dropping\&. Here are the current features of MultiMC\&.
+UltimMC is a custom launcher for Minecraft that allows you to easily manage multiple installations of Minecraft at once\&. It also allows you to easily install and remove mods by simply dragging and dropping\&. Here are the current features of UltimMC\&.
```

Comment:
Add here note about light-fork of MultiMC with cracked accounts and accounts from ely.by